packages/nodes-base/test/nodes/Helpers.ts

```diff
@@ -373,6 +386,9 @@ export const workflowToTests = (workflowFiles: string[]) => {
 				);
 			}
 		});
+
+		if (process.env.NODE_ENV === 'benchmark') workflowData.pinData = {};
```

Comment:
we use `pinData` for assertions. does this change mean that we disable those assertions when running nodes tests for benchmarking?

Reply:
Benchmarks use `executeWorkflow` but do not use `equalityTest` so they sidestep all Jest assertions. 

This line specifically I added because a handful of workflows (MoveBinaryData, QuickChart) were getting caught by the pindata check right after this line, because they have their own implementation of `testWorkflows`.